src/app/components/profile/profile.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, OnInit} from '@angular/core';
+import {AuthService} from '@shared/services/auth.service';
+import {MatDialog} from '@angular/material/dialog';
+import {ConfirmDeleteProfileComponent} from '../confirm-delete-profile/confirm-delete-profile.component';
+import {Router} from '@angular/router';
+import {MatSnackBar} from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'app-profile',
+  templateUrl: './profile.component.html',
+  styleUrls: ['./profile.component.scss'],
+})
+export class ProfileComponent implements OnInit {
+  name: string | undefined;
```

Comment:
Do not need to duplicate all user fields in the component, use only the user object and get fields in the template.

Reply:
I don`t understand

Reply:
component:
`user: User;`
template:
```
{{ user.name }}
{{ user.email }}
```

Reply:
Done